config/application.rb

```diff
@@ -49,5 +49,6 @@ class Application < Rails::Application
     config.active_model.i18n_customize_full_message = true
 
     config.email_gateway = Gateways::EmailGateway
+    config.certificate_repository = Gateways::CertificateRepositoryS3
```

Comment:
Is this necessary? It doesn't seem to be overloaded in any of the other environments

Reply:
Might be nice to swap it out for another implementation in a similar way to the email gateway - but the whole s3 repo could potentially go, so maybe not needed..